package.json

```diff
@@ -30,11 +30,13 @@
     "Conduction"
   ],
   "dependencies": {
+    "@mui/icons-material": "^5.3.0",
```

Comment:
Willen we een dependency op mui icons hebben? Voor iconen gebruiken wij volgens mij font-awesome

Reply:
Ah, ik zie het nu. Ik dacht dat we júíst op MUI wilde zitten. Ik pas het aan 👍🏼